pkg/validator/server.go

```diff
@@ -74,14 +82,20 @@ type ValidatingWebhookServer struct {
 }
 
 // New returns a ValidatingWebhookServer with the defaultValidators that were previously registered.
-func New() *ValidatingWebhookServer {
+func New(name string, port int, certificater certificate.Certificater, kubeClient kubernetes.Interface, stop <-chan struct{}) (*ValidatingWebhookServer, error) {
```

Comment:
To avoid confusing the name of the webhook config and the name of the webhook
```suggestion
func New(webhookConfigName string, port int, certificater certificate.Certificater, kubeClient kubernetes.Interface, stop <-chan struct{}) (*ValidatingWebhookServer, error) {
```

Reply:
done!